cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -337,6 +335,20 @@ private void assignNetworkProperties(Context context) {
                 context.nodeBreaker()
                         ? NETWORK_PS_CGMES_MODEL_DETAIL_NODE_BREAKER
                         : NETWORK_PS_CGMES_MODEL_DETAIL_BUS_BRANCH);
+        PropertyBags modelProfiles = context.cgmes().modelProfiles();
+        String fullModel = "FullModel";
+        modelProfiles.sort(Comparator.comparing(p -> p.getId(fullModel)));
+        for (PropertyBag modelProfile : modelProfiles) {
+            if (modelProfile.getId(fullModel).equals(context.network().getId())) {
+                continue;
+            }
+            String profile = CgmesNamespace.getProfile(modelProfile.getId("profile"));
+            if (profile != null && !"EQ_OP".equals(profile) && !"SV".equals(profile)) {
```

Comment:
Maybe we should comment why we ignore the `EQ_OP` and `SV` profiles here

Reply:
Or just let them be stored also as properties... We won't use them in the export, but they may be relevant for other purposes ?

Reply:
I would prefer not to import them if not useful for now. When we use extensions, we will import all the profiles.

Reply:
comments added